assets/charts/control-plane/kubelet/templates/kubelet-ds.yaml

```diff
@@ -125,10 +125,6 @@ spec:
         - name: etc-resolv
           mountPath: /etc/resolv.conf
           readOnly: true
-        # This is mounted so that storage works fine.
```

Comment:
IMO this should be committed together with kubelet image change, otherwise checking out this commit will cause a regression.

Alternatively, please swap the commit order so we first replace the image, then we remove the bind mount. This won't create a regressing revision.